app/helpers/facets_helper.rb

```diff
@@ -60,4 +60,22 @@ def facet_item_component_class(facet_config)
     default_component = facet_config.pivot ? Blacklight::FacetItemPivotComponent : PsulFacetItemComponent
     facet_config.fetch(:item_component, default_component)
   end
+
+  def campus_facet_all_online_links
```

Comment:
Also, I wasn't too sure if this was the right place to put this method.  It's related to the facets, but isn't actually in the facets drop-down.

Reply:
I think it makes sense here 👍🏻